pkg/setting/setting_plugins.go

```diff
@@ -52,6 +52,7 @@ func (cfg *Cfg) readPluginSettings(iniFile *ini.File) error {
 			}
 			cfg.InstallPlugins[i] = InstallPlugin{id, v}
 		}
+		cfg.InstallPluginsBlock = pluginsSection.Key("preinstall_block").MustBool(false)
```

Comment:
What about flipping it instead? I feel like this is more intuitive perhaps.
```suggestion
		cfg.InstallPluginsAsync = pluginsSection.Key("preinstall_async").MustBool(true)
```